tempto-core/src/main/java/com/teradata/tempto/fulfillment/table/jdbc/tpch/JdbcTpchTableDefinitions.java

```diff
@@ -29,8 +30,9 @@
     public static final double DEFAULT_SCALE_FACTOR = 0.01;
     public static final ImmutableList<JDBCType> NATION_TYPES = ImmutableList.of(BIGINT, VARCHAR, BIGINT, VARCHAR);
 
+    @RepositoryTableDefinition
     public static final RelationalTableDefinition NATION_DDL =
```

Comment:
just call this NATION to match the naming scheme of the other table definitions.